packages/components/src/internal/components/domainproperties/DateTimeFieldOptions.tsx

```diff
@@ -214,6 +225,8 @@ export const DateTimeFieldOptions: FC<DateTimeFieldProps> = memo(props => {
                 {setting.isDate && (
                     <div className="col-xs-3">
                         <SelectInput
+                            allowCreate
+                            isValidNewOption={isValidNewOption}
```

Comment:
could `isValidAltDateTimeFormatOptions` just be used directly here? just curious if the useCallback is needed since it just passes the single parm